src/assets/wise5/services/dataExportService.ts

```diff
@@ -2,13 +2,18 @@ import { HttpClient, HttpParams } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { ConfigService } from './configService';
 import { TeacherDataService } from './teacherDataService';
+import { UtilService } from './utilService';
 
 @Injectable()
 export class DataExportService {
+  studentEventsByWorkgroupId: any;
+  teacherEventsByUserId: any;
```

Comment:
Should these be named eventsByWorkgroupId and eventsByUserId? There is no StudentEvent and TeacherEvent, only Event.